source/main/gui/panels/GUI_GameConsole.cpp

```diff
@@ -235,183 +236,219 @@ void Console::eventCommandAccept(MyGUI::Edit* _sender)
         // discard the empty message
         return;
     }
-        Ogre::StringVector args = StringUtil::split(msg, " ");
```

Comment:
This formatting makes the whole diff unreadable. Again - do a separate 📐 revision for these pls.